sky/backends/backend_utils.py

```diff
@@ -975,8 +975,8 @@ def _add_auth_to_cluster_config(cloud: clouds.Cloud, cluster_config_file: str):
     common_utils.dump_yaml(cluster_config_file, config)
 
 
-def get_run_timestamp() -> str:
-    return 'sky-' + datetime.now().strftime('%Y-%m-%d-%H-%M-%S-%f')
+# Reuse the sky_logging for consistency and avoid circular import.
+get_run_timestamp = sky_logging.get_run_timestamp
```

Comment:
Seems unused. remove?

Reply:
There are multiple places in the code using `run_timestamp = backend_utils.get_run_timestamp()`. This ensures they still work.

Reply:
I see - can we update those references to directly use sky_logging. get_run_timestamp() instead?

Reply:
Done